src/main/resources/elasticsearch.template.properties

```diff
@@ -0,0 +1,36 @@
+# ============ elasticsearch configuration for picturesafe-search =============
+#
+# picturesafe-search requires a configuration file named 'elasticsearch.properties' in the classpath,
+# in which the following default parameters can be overwritten.
```

Comment:
Sound a bit strange. What about "You can use the following default parameters by leaving them commented or, if you want to change settings, uncomment lines and change the values."?

Reply:
Hm, this is a template file inside the lib, which may change with a release.
I would at least leave the first sentence and then insert yours. Ok?